src/sentry/static/sentry/app/views/organizationGroupDetails/reprocessingForm.tsx

```diff
@@ -38,60 +40,55 @@ class ReprocessingForm extends React.Component<Props> {
           <FeatureBadge type="alpha" />
         </Header>
         <Body>
-          <Alert type="warning">
-            {t(
-              'Reprocessing is a preview feature. Please carefully review the limitations and implications!'
-            )}
-          </Alert>
-
           <p>
             {t(
-              'You can choose to reprocess issues to apply new debug files and updated grouping configuration. While reprocessing is in preview, keep the following limitations in mind:'
+              'Reprocessing applies any new debug files or grouping configuration to an Issue. Right now, this is a preview feature. So before you give it a try, you should probably consider these impacts:'
             )}
           </p>
 
-          <ul>
-            <li>
+          <StyledList>
+            <ListItem>
               {tct(
-                'Sentry [strong:creates new events and deletes this issue.] This may temporarily affect event counts in Discover and the Issue Stream.',
+                '[strong:Creates new events and deletes the old issue.] This may temporarily affect event counts in both Discover and the Issue Stream.',
                 {strong: <strong />}
               )}
-            </li>
-            <li>
+            </ListItem>
+            <ListItem>
               {tct(
-                'Depending on the number of events, [strong:reprocessing can take several minutes.] Once started, Sentry opens a view on the Issue Stream that shows the reprocessed issues.',
+                '[strong:Reprocessing may take a while.] See progress as it happens in the Issue Stream.',
```

Comment:
Technically, we cannot show progress at the moment. Instead, we just open a view that has to be refreshed manually.

Reply:
I think that ambiguity is okay here.

Reply:
We have to be very careful as this sets wrong expectations on a feature that is already quite constrained.